content/contribute/index.md

```diff
@@ -0,0 +1,133 @@
+chances are that you probably can help a lot of people with their questions.
+
+We mostly do support via the forum we have here: https://discuss.ipfs.io/
+
+### Testing
+
+As we're continously improving IPFS everyday, mistakes can happen and we release
+something that could work much better or simply doesn't work. If you like to
+dig into edge-cases or writing testing scenarios, getting dirty with all our
+testing infrastructure would be a wet dream for you.
+
+Mainly, CI work is happening over at https://github.com/ipfs/jenkins and larger
+scale tests are planned to be happening over at https://github.com/ipfs/kubernetes-ipfs
+
+
+### Design
+
+Currently, there is no fulltime designers working with IPFS and it's sister-projects.
+Although it would be nice to have, we still have many smaller pieces that can be
+contributed by you as a outsider!
+
+**We currently don't have a single place for this. If you'd like to start it, please let us know**
+
+### Issues / Triaging
+
+With lots of code, comes lots of Github Issues. We need YOU to help with
+organizing all of this in some manner. We don't quite yet have any proper resources
+for getting started with this.
+
+**We currently don't have a single place for this. If you'd like to start it, please let us know**
+
+### Community
+
+If interacting with people is your favorite thing to do in this world, IPFS and
+co are always happy to help you organize events and/or workshops to teach IPFS.
+
+We do have a repository (https://github.com/ipfs/community) for organize community
+events, and would love your help to have meetups in more locations or make the
+existing ones more regular.
+
+
+### Applications
+
+IPFS is built so others can build applications using it! Building applications and
+services using IPFS is a excellent way of finding use cases where IPFS quite
+doesn't the job as great as it could, or to uncover bugs/inefficiences.
+
+A place to get started is to look at our awesome-ipfs list, and build anything
+that you think is missing. If you're unsure about something, you're welcome to
+create a issue to get help or feedback on your specific problem/idea.
+
+- https://github.com/ipfs/awesome-ipfs
+- https://github.com/ipfs/notes
+- https://github.com/ipfs/apps
+- https://discuss.ipfs.io/
+
+### Protocol Design
+
+Since IPFS is a protocol in the end, we always welcome ideas and feedback on how
+to improve it. Most of the work would happen within the specifications or ipfs/notes
+
+- https://github.com/ipfs/specs
+- https://github.com/libp2p/specs
+- https://github.com/ipld/specs
+- https://github.com/multiformats/specs
+- https://github.com/ipfs/notes
+
+### Research
+
+Protocol Labs is in the ground and bottom a research lab, where YOUR idea can
+become real, if it can have a real impact on the world. If you're interested in
+contributing to the research, please reach out to research@ipfs.io or research@protocol.ai
```

Comment:
Should these be `mailto:` links, or are we trying to avoid that?

Reply:
Yeah, `mailto:` sounds good, think it was just because of laziness I didn't add it. Can't find a reason why we would avoid it.